src/components/diagnosticReportView/DiagnosticReportCard.tsx

```diff
@@ -214,9 +236,10 @@ const DiagnosticReportCard: React.FC<DiagnosticReportCardProps> = ({
           </div>
           <Tabs value={currentTab} onChange={handleTabChange}>
             <Tab label="General" id="simple-tab-0" disableFocusRipple />
-            <Tab label="Named Faults" id="simple-tab-1" disabled={modelType !== 'CRUSTAL'} disableFocusRipple />
-            <Tab label="Regional Solutions" id="simple-tab-2" disabled={modelType !== 'CRUSTAL'} disableFocusRipple />
-            <Tab label="Hazard Charts" id="simple-tab-3" disabled={!hazardId.length} disableFocusRipple />
+            <Tab label="Regional Solutions" id="simple-tab-1" disabled={modelType !== 'CRUSTAL'} disableFocusRipple />
+            <Tab label="Named Faults" id="simple-tab-2" disabled={modelType !== 'CRUSTAL'} disableFocusRipple />
+            <Tab label="Parent Fault" id="simple-etab-3" disabled={modelType !== 'CRUSTAL'} disableFocusRipple />
```

Comment:
@0qy can tab name be pluralised please => "Parent Faults". I know it only displays 1 PF currently, but the subject matter is all the "parent faults"